tests/unit_tests/libs/helpers/text_helpers.py

```diff
@@ -0,0 +1,46 @@
+from collections import Counter
+import pytest
+import random
+
+from mindsdb_native.libs.helpers.text_helpers import (
+    get_language_dist,
+    analyze_sentences
+)
+
+
+def test_language_analysis():
+    WORDS = {
+        'en': ['becuase', 'these', 'first', 'work', 'interpret', 'call', 'think'],
+        'ru': ['только', 'говорить', 'когда', 'человек', 'быть', 'первый', 'осень'],
+        'de': ['führen', 'Stelle', 'heißen', 'konnten', 'schlimm', 'mögen', 'Nähe'],
+    }
+
+    sent_size = 7
+    num_sents = 10
+
+    for lang, words in WORDS.items():
+        sentences = [random.sample(words, sent_size) for _ in range(num_sents)]
+
+        nr_words, word_dist, nr_words_dist = analyze_sentences(' '.join(sent) for sent in sentences)
```

Comment:
All good. Would be better if `analyze_sentences` had a more intuitive name matching what it does

Reply:
it kinds of does too many things to name it something more specific than "analyze". If you have a name suggestion, we can change it later.